constants.ts

```diff
@@ -0,0 +1,5 @@
+// Canonical URL
+export const canoncialUrl: string =
+	process.env.NEXT_PUBLIC_TARGET_ENV === 'production'
```

Comment:
You need to change `production` to `prod`. The 3 states of target_env are `prod`, `staging`, and `dev`